packages/syntax-core/src/colors/badgeBackgroundColor.ts

```diff
@@ -0,0 +1,23 @@
+import { type BadgeColor } from "../constants";
+import styles from "./colors.module.css";
+
+export default function badgeBackgroundColor(
```

Comment:
Not a must but feel free to inline this function in `Badge.tsx` since we only use it within that file. In general it makes sense to abstract it out if it's used in 2+ places